decentraland/unity-renderer
# feat: channels search pagination by token

## What does this PR change?

Changes the way pagination works when searching channels. It is required a `string paginationToken` instead of a `skip` parameter.

## How to test the changes?

1. Open the channel browser
2. All channels should be loaded
3. Scroll down to load more
4. Use the searchbar and type a name

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
